pkg/manifests/manifests.go

```diff
@@ -259,6 +288,42 @@ func DaemonSet(component string) (*appsv1.DaemonSet, error) {
 	if !ok {
 		return nil, fmt.Errorf("unexpected type, got %t", obj)
 	}
+
+	for i := range ds.Spec.Template.Spec.Containers {
+		c := &ds.Spec.Template.Spec.Containers[i]
+		if c.Name == containerNameRTE {
+			c.Image = images.ResourceTopologyExporterImage
+			c.Command = []string{
```

Comment:
this is better than before but I'm not really sold about it. But we can surely try and evolve incrementally.

Reply:
So let's go with this, we will improve later if needed once we have a clearer picture